ts/webui/src/static/model/localStorage.ts

```diff
@@ -0,0 +1,41 @@
+export interface StorageFormat {
+    expire: number;
+    time: number;
+    value: string;
+}
+
+export const getValue = (key): null | string => {
+    const val = localStorage.getItem(key);
+    if (!val) {
+        return null;
+    }
+    const data = JSON.parse(val) as StorageFormat;
+    if (Date.now() - data.time > data.expire) {
```

Comment:
This is not a correct expire. This is "expire on access". It means that it won't expire if nobody tries to access it.

Reply:
@liuzhe-lz we plan to delete expire time in v2.9.1. Do you agree with us?